lib/grant.js

```diff
@@ -141,6 +143,8 @@ function checkClient (done) {
       return done(error('invalid_client', 'Client credentials are invalid'));
     }
 
+    // Expose validated client
+    self.scope = client.scope;
```

Comment:
@nunofgs: Just to be on the same page here, is this meant for preloading the scopes the client *may* solicit, or *default* scope keys?